.github/workflows/build.yml

```diff
@@ -9,39 +9,41 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
+        java: [11, 17, 20]
```

Comment:
why not yet 21 ?

Reply:
It's stated in https://github.com/spotbugs/spotbugs/issues/2598: 
> Originally I wanted jdk 21 to show case it working due to various user inquires but fact of the matter is gradle doesn't support it yet until 8.4 and still won't with kotlin which means we cannot actually show that. So that is where jdk 20 comes into place.